fetch.bs

```diff
@@ -4820,17 +4963,24 @@ these steps:
 
         <ol>
          <li><p>Let <var>bytes</var> be the transmitted bytes.
+         <li><p>Let <var>timingInfo</var> be the <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
 
          <li><p>Let <var>codings</var> be the result of <a>extracting header list values</a> given
          `<code>Content-Encoding</code>` and <var>response</var>'s <a for=response>header list</a>.
 
+         <li><p>Set <var>timingInfo</var>'s <a for="fetch timing info">encoded body size</a> to the result of
+         adding <var>timingInfo</var>'s <a for="fetch timing info">encoded body size</a> and <var>bytes</var>.
```

Comment:
Yes it should have the byte size regardless of content-length. At least, it's not specified otherwise.
There doesn't seem to be a test specifically for this.